src/ru/nudemoon/src/eu/kanade/tachiyomi/extension/ru/nudemoon/Nudemoon.kt

```diff
@@ -159,12 +159,12 @@ class Nudemoon : ParsedHttpSource(), ConfigurableSource {
 
     override fun mangaDetailsParse(document: Document): SManga {
         val manga = SManga.create()
-        val infoElement = document.select("table.news_pic2").first()!!
-        manga.title = document.select("h1").first()!!.text().substringBefore(" / ").substringBefore(" №")
-        manga.author = infoElement.select("a[href*=mangaka]").text()
-        manga.genre = infoElement.select("div.tag-links a").joinToString { it.text() }
+        val infoElement = document.selectFirst("table.news_pic2")
+        manga.title = document.selectFirst("h1")?.text()?.substringBefore(" / ")?.substringBefore(" №") ?: "No title"
```

Comment:
`title` is required for manga and should not use a fallback value.
```suggestion
        manga.title = document.selectFirst("h1")!!.text().substringBefore(" / ").substringBefore(" №")
```

Reply:
Sorry not seen button just do myself